solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreakerManager.java

```diff
@@ -39,26 +45,21 @@
  * NOTE: The current way of registering new default circuit breakers is minimal and not a long term
  * solution. There will be a follow up with a SIP for a schema API design.
  */
-public class CircuitBreakerManager implements PluginInfoInitialized {
+public class CircuitBreakerManager implements ResourceLoaderAware, PluginInfoInitialized {
```

Comment:
question 2 of 2: If (and that is an if) the circuit breaker manager no longer needs to be configurable then perhaps the ` PluginInfoInitialized` no longer needs to be implemented here?

```suggestion
public class CircuitBreakerManager implements ResourceLoaderAware {
```